### ISSUE TITLE
Github Desktop constantly "installing helper tool"

### ISSUE THREAD

**Issue by oguzyildiz1991** [desktop/desktop#10777]
### Describe the bug

Hi, I'm getting the prompt for "Github Desktop is trying to helper tool" and my mac is asking for password. Whether I enter my password or not, nothing happens, and I just get the same prompt few hours later.

In the About menu, I clicked Check for updates and there's no update to install.

### Version & OS

Version 2.5.6 
MacOS Catalina 10.15.7

### Steps to reproduce the behavior

1. Happens by itself

### Expected behavior

Stop unnecessary prompts

### Actual behavior

I'm getting the prompt for "Github Desktop is trying to helper tool" and my mac is asking for password. Whether I enter my password or not, nothing happens, and I just get the same prompt few hours later.

### Screenshots

I can get a screenshot next time it happens

### Logs

I don't see the "Show Logs" button under the Help menu

**Comment by steveward** [comment]
Thanks for the report. The steps mentioned in https://github.com/desktop/desktop/issues/3616#issuecomment-400371047 should hopefully get this resolved for you. Let me know if that does not help.

**Comment by matthewfx** [comment]
I have been experiencing the same issue since I bought a new MacMini M1 (Big Sur).

**Comment by mitch-zink** [comment]
I'm having the same issue. I keep entering my creds but it doesn't stop.

**Comment by klo1991qa** [comment]
Thx

**Comment by ShinanSun** [comment]
I am having the same issue. Adding helper over and over again non-stop

**Comment by Everything-In-Sync** [comment]
Same issue, it just popped up again prompting me to find this thread

**Comment by benhorowitz** [comment]
Same issue here

**Comment by ebellumat** [comment]
Same here.

**Comment by chrisdavis-ae** [comment]
Ditto

**Comment by garyvaughanjr** [comment]
How is this closed when it is still a problem?

**Comment by garyvaughanjr** [comment]
```
gvz@s-007 web % ls -l /Applications | grep "GitHub Desktop"
drwxr-xr-x  3 root         wheel   96 Jun  1 01:23 GitHub Desktop.app

gvz@s-007 web % sudo chown gvz "/Applications/Github Desktop.app"

gvz@s-007 web % ls -l /Applications | grep "GitHub Desktop"         
drwxr-xr-x  3 gvz  wheel   96 Jun  1 01:23 GitHub Desktop.app
```

**Comment by imsangram** [comment]
Still getting this issue!

**Comment by HedvigS** [comment]
I'm having the same issue. I'm going to try the steps suggested above.

**Comment by HedvigS** [comment]
> Thanks for the report. The steps mentioned in [#3616 (comment)](https://github.com/desktop/desktop/issues/3616#issuecomment-400371047) should hopefully get this resolved for you. Let me know if that does not help.

The issue thread linked above has 66 comments.

What are the instructions we ought to follow?

**Comment by HedvigS** [comment]
It seems to be this:

> To everyone in this thread still encountering issues, check the permissions of the app are what you expect:
> 
> ```shell
> ~/ ls -l /Applications | grep "GitHub Desktop"
> drwxr-xr-x   3 shiftkey  staff   96 25 Jun 15:57 GitHub Desktop.app
> ```
> 
> If `root` owns this for some reason, or an account that you weren't expecting, you might need to `chown` it to change the ownership.

I don't know what it means to "`chown`". 

I get

```
 ~ % ~/ ls -l /Applications | grep "GitHub Desktop"
zsh: permission denied: /Users/username
```

**Comment by h-bomb** [comment]
I had the same issue. In my case, on my Mac, I had created a new profile (account) and just copied over applications, files, etc. to my new account. The app had the other account's permissions (**Privileges**) set on the app file. I added the new account and used the same **Privilege** settings as the other account. The dialog no longer displays. (So, similar to what others posted above - i.e., a permission-based issue.)
